openqabot/approver.py

```diff
@@ -35,22 +35,26 @@ def _mi2str(inc: IncReq) -> str:
 def _handle_http_error(e: HTTPError, inc: IncReq) -> bool:
     if e.code == 403:
         log.info(
-            "Received '%s'. Request %s likely already approved, ignoring"
-            % (e.reason, inc.req)
+            "Received '%s'. Request %s likely already approved, ignoring",
+            e.reason,
+            inc.req,
         )
         return True
-    elif e.code == 404:
+    if e.code == 404:
```

Comment:
[if/else] To be carefully reviewed

Reply:
that's fine because the previous if concludes with a return so the code execution would never execute the above branch and reach here. So the change makes sense